Lib/idlelib/configdialog.py

```diff
@@ -672,18 +674,18 @@ def create_page_general(self):
             frame
                 frame_run: LabelFrame
                     startup_title: Label
-                    (*)radio_startup_edit: Radiobutton - startup_edit
-                    (*)radio_startup_shell: Radiobutton - startup_edit
+                    (*)startup_editor_on: Radiobutton - startup_edit
+                    (*)startup_shell_on: Radiobutton - startup_edit
                 frame_save: LabelFrame
                     run_save_title: Label
-                    (*)radio_save_ask: Radiobutton - autosave
-                    (*)radio_save_auto: Radiobutton - autosave
+                    (*)save_ask_on: Radiobutton - autosave
+                    (*)save_auto_on: Radiobutton - autosave
```

Comment:
Same here as above, I was hoping to simplify the names to `autosave` and the choices to `prompt` and `noprompt`.